Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM mfdz/opentripplanner:2022-03-28-13_44 AS otp
+# syntax=docker/dockerfile:1.2
 
-# defined empty, so we can access the arg as env later again
+ARG OTP_TAG=2022-05-23-18_04
+ARG OTP_IMAGE=mfdz/opentripplanner
+
+FROM $OTP_IMAGE:$OTP_TAG AS otp
 
-# GTFS der VBB
-ARG gtfs_url=http://vbb.de/vbbgtfs
+# defined empty, so we can access the arg as env later again
 
-# GTFS Daten von delfi (#BBNAV-71)
-# ARG gtfs_url=https://gtfs.mfdz.de/DELFI.BB.gtfs.zip
+# GTFS Daten vom VBB
+ARG gtfs_url=https://www.vbb.de/fileadmin/user_upload/VBB/Dokumente/API-Datensaetze/gtfs-mastscharf/GTFS.zip
 ENV GTFS_URL=$gtfs_url
 
+# GTFS Daten von fahrgemeinschaft mifaz
+# URL ist hinterlegt in GITHUB Secrets: GTFS_CARPOOL_URL
+#RUN --mount=type=secret,id=GTFS_CARPOOL_URL export GTFS_CARPOOL_URL=$(cat /run/secrets/GTFS_CARPOOL_URL) && curl -LJO $GTFS_CARPOOL_URL
```

Comment:
Is this still needed? If not, let's remove it.

Reply:
Removed.